packages/protoc-gen-connect-query-react/src/generateTs.ts

```diff
@@ -102,38 +103,12 @@ const generateServiceFile =
           importHookFrom,
         );
 
-        f.print(
```

Comment:
Removed prettier formatting on this since it's much easier to read without.

Reply:
I agree. Prettier is not made for formatting code generators 🙂